pkg/operator/sync_apiservice_v4_00.go

```diff
@@ -63,7 +63,7 @@ func syncAPIServiceController_v4_00_to_latest(c serviceCAOperator, operatorConfi
 	}
 
 	// TODO create a new configmap whenever the data value changes
-	_, configMapModified, err := manageAPIServiceConfigMap_v4_00_to_latest(c.corev1Client, c.eventRecorder, operatorConfig)
+	_, configMapModified, err := manageAPIServiceConfigMap_v4_00_to_latest(c.corev1Client, c.eventRecorder)
```

Comment:
Keep passing this through and add TODO to combine CLI config and allow overrides with unsupported config overrides.

Reply:
follow-up ok?